sos/plugins/__init__.py

```diff
@@ -840,7 +983,7 @@ def get_cmd_output_now(self, exe, suggest_filename=None,
     def is_module_loaded(self, module_name):
         """Return whether specified moudle as module_name is loaded or not"""
         if len(grep("^" + module_name + " ", "/proc/modules")) == 0:
-            return None
+            return False
         else:
             return True
```

Comment:
Can't we simplify the method to single:

    return (len(grep("^" + module_name + " ", "/proc/modules")) > 0)

?

Reply:
I think we can!